drivers/storage/portworx/components_test.go

```diff
@@ -2963,7 +2921,7 @@ func TestCSIInstallWithNewerCSIVersion(t *testing.T) {
 
 	expectedCR := testutil.GetExpectedClusterRole(t, "csiClusterRole_k8s_1.13.yaml")
 	actualCR := &rbacv1.ClusterRole{}
-	err = testutil.Get(k8sClient, actualCR, csiClusterRoleName, "")
+	err = testutil.Get(k8sClient, actualCR, "px-csi", "")
```

Comment:
nit feel free to ignore: you could make all of them global constants in pkg/components
So that you don't need to keep changing the tests if a name changes.